<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite parallel="tests" name="Failed suite [Suite]" guice-stage="DEVELOPMENT" verbose="0">
  <test thread-count="5" parallel="tests" name="Submit Order Test(failed)" verbose="0">
    <classes>
      <class name="danish.tests.SubmitOrderTest">
        <methods>
          <include name="launchApplication"/>
          <include name="OrderHistoryTest"/>
          <include name="close"/>
          <include name="submitOrder" invocation-numbers="0 1"/>
        </methods>
      </class> <!-- danish.tests.SubmitOrderTest -->
    </classes>
  </test> <!-- Submit Order Test(failed) -->
  <test thread-count="5" parallel="tests" name="Error Validations Test(failed)" verbose="0">
    <classes>
      <class name="danish.tests.ErrorValidationsTest">
        <methods>
          <include name="productErrorValidation"/>
          <include name="launchApplication"/>
          <include name="loginErrorValidation"/>
          <include name="close"/>
        </methods>
      </class> <!-- danish.tests.ErrorValidationsTest -->
    </classes>
  </test> <!-- Error Validations Test(failed) -->
</suite> <!-- Failed suite [Suite] -->
